plugin/Model/InputModelBase.php

```diff
@@ -102,6 +102,9 @@ abstract class InputModelBase
 	/** @var array */
 	private $_labelOnlyValues = [];
 
+	/** @var string */
+	private $_descriptionTextHTML = null;
```

Comment:
Better initialize as empty string, then add the `string` return type to the getter

Reply:
ok I will update